deploy/addon/config-provider.md

```diff
@@ -1,28 +1,71 @@
 ---
-title: Configuration provider
-shortdesc: Share configuration between apps
-tags:
-- addons
-keywords:
+title: Configuration provider shortdesc: Share configuration between apps tags:
+
+- addons keywords:
```

Comment:
You removed the tags ?

Reply:
I blame my WebStorm config, it applies some weird formatting... :-(